frontend/packages/knative-plugin/src/utils/knative-topology-utils.ts

```diff
@@ -119,18 +119,29 @@ export const getKnativeTopologyNodeItem = (
     const configurations = getOwnedResources(resource, resources.configurations.data);
     const configUidData = _.get(configurations[0], ['metadata', 'uid']);
     const ChildData = _.filter(resources.revisions.data, {
-      metadata: { ownerReferences: [{ uid: configUidData }] },
+      metadata: {
+        ownerReferences: [{ uid: configUidData }],
+      },
     });
     _.forEach(ChildData, (c) => {
-      children.push(_.get(c, ['metadata', 'uid']));
+      const nameRev = _.get(c, ['metadata', 'name']);
+      const labelRev = _.get(c, ['metadata', 'labels', 'app.openshift.io/instance']);
+      const uidRev = _.get(c, ['metadata', 'uid']);
+      children.push(uidRev);
+      nodes.push({
+        id: uidRev,
+        type: nodeType.Revision,
+        name: labelRev || nameRev,
+      });
```

Comment:
you can use the `getTopologyNodeItem` utility  from topology-utils and pass type and children as parameters?